MSAL/src/native_auth/controllers/reset_password/MSALNativeAuthResetPasswordController.swift

```diff
@@ -453,13 +460,15 @@ final class MSALNativeAuthResetPasswordController: MSALNativeAuthBaseController,
             case .succeeded:
                 stopTelemetryEvent(event, context: context)
```

Comment:
Telemetry event should not be stopped here as it will be at line 146

Reply:
Done (same as [here](https://github.com/AzureAD/microsoft-authentication-library-for-objc/pull/1914#discussion_r1400963221))